lib/src/github_sign_in_page.dart

```diff
@@ -35,18 +35,20 @@ class _GitHubSignInPageState extends State<GitHubSignInPage> {
     return Scaffold(
         appBar: new AppBar(
           title: Text(widget.title),
+          centerTitle: true,
         ),
         body: WebView(
           initialUrl: widget.url,
           userAgent: widget.userAgent ?? _userAgentMacOSX,
+          javascriptMode: JavascriptMode.unrestricted,
           onPageFinished: (url) {
-            if (url.startsWith(widget.redirectUrl)) {
```

Comment:
Hi is checking "error=" before redirect URL will solve anything?

Reply:
I reversed the conditions because when you click on Canceled, the url is that of the redirectUrl but with "error=" inside
So even if there is "error=" in the url, if it starts redirectUrl it takes that as a success

So now it first checks if there is no error even on the firebase redirectUrl

Reply:
This is the url when click on Cancel : 
https://l2t-flutter.firebaseapp.com/__/auth/handler?error=access_denied&error_description=The+user+has+denied+your+application+access.&error_uri=https%3A%2F%2Fdocs.github.com%2Fapps%2Fmanaging-oauth-apps%2Ftroubleshooting-authorization-request-errors%2F%23access-denied

It starts well with the redirectUrl but there is error in query parameter